docs/specs/aerogear-client-pipe/rest-adapter.asciidoc

```diff
@@ -1,6 +1,7 @@
 ---
```

Comment:
I can't see this document linked from the Site, is it? If it should be removed, let's remove it, if not, let's link it.